src/components/pages/Onboarding/Onboarding.tsx

```diff
@@ -221,11 +221,9 @@ class OnboardingView extends React.Component<OnboardingProps, OnboardingState> {
   };
 
   _renderOnboardingStep() {
-    if (!this.props.isLoggedIn) {
+    if (this.props.hasValidInviteToken) {
```

Comment:
I'm confused by this -- shouldn't we only redirect if they're not logged-in? If the user is loggedIn and has a validInviteToken, won't this still redirect them to Login?

Reply:
ah you're right. The Onboarding component was being mistakenly mounted due to issue 3 as listed above so I prematurely changed this. But we should only reach the Onboarding component while not logged in if the user does have an invite token. Changing this back